packages/api/src/command/medical/document/document-webhook.ts

```diff
@@ -39,13 +39,14 @@ export const processPatientDocumentRequest = async (
   patientId: string,
   whType: MAPIWebhookType,
   status: MAPIWebhookStatus,
-  documents?: DocumentReferenceDTO[]
+  documents?: DocumentReferenceDTO[],
+  cxPayload?: object
```

Comment:
shouldn't this be required?

Reply:
No because the payload is optional for the /document/query endpoint right?

Reply:
💯